gigantum/gigantum-client
# (#1557) Fixes summary showing negative value

## Pull Request for Issue #1557

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
Refactors type and summary components. Fixes negative summary value showing.


**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
Upload src folder from gigantum-client ui folder and refresh to see if a negative to download value appears.
